package/llvm/llvm.mk

```diff
@@ -4,16 +4,18 @@
 #
 ################################################################################
 
-# LLVM, Clang and lld should be version bumped together
-# batocera  bump to 13.0.1
-LLVM_VERSION = 13.0.1
+# LLVM, Clang, libclc and lld should be version bumped together
+# batocera  bump to 14.0.6
+LLVM_VERSION = 14.0.6
 LLVM_SITE = https://github.com/llvm/llvm-project/releases/download/llvmorg-$(LLVM_VERSION)
-LLVM_SOURCE = llvm-$(LLVM_VERSION).src.tar.xz
+LLVM_SOURCE = llvm-project-$(LLVM_VERSION).src.tar.xz
 LLVM_LICENSE = Apache-2.0 with exceptions
 LLVM_LICENSE_FILES = LICENSE.TXT
 LLVM_CPE_ID_VENDOR = llvm
 LLVM_SUPPORTS_IN_SOURCE_BUILD = NO
 LLVM_INSTALL_STAGING = YES
+# Main CMakeLists.txt in llvm subfolder
+LLVM_SUBDIR = llvm
```

Comment:
personnally, i add #batocera for each block, to be sure this kind of line doesn't disappear in merges